test/vectors/issuance/1.issue-credential_propose.json

```diff
@@ -0,0 +1,10 @@
+{
+    "type":"https://didcomm.org/issue-credential/3.0/propose-credential",
+    "from":"did:peer:1zQmYkYz1brNfrAGq15A3YSxSFsEhmaG9TBzKx8wMyDFR261?initialState=W3siY2hhbmdlIjoiZXlKQVkyOXVkR1Y0ZENJNld5Sm9kSFJ3Y3pvdkwzZDNkeTUzTXk1dmNtY3Zibk12Wkdsa0wzWXhJbDBzSW1sa0lqb2laR2xrT25CbFpYSTZNWHBSYlZscldYb3hZbkpPWm5KQlIzRXhOVUV6V1ZONFUwWnpSV2h0WVVjNVZFSjZTM2c0ZDAxNVJFWlNNall4SWl3aWRtVnlhV1pwWTJGMGFXOXVUV1YwYUc5a0lqcGJleUpqYjI1MGNtOXNiR1Z5SWpvaUlpd2lhV1FpT2lJamEyVjVMVEVpTENKd2RXSnNhV05MWlhsQ1lYTmxOVGdpT2lJM2QyTjJZa0prUmtGS1dGVmFVRGhwUVhacFYweHpSazV1Y201YWJWSk5OelY1VTFodlFsRmtjek5PU0NJc0luUjVjR1VpT2lKRlpESTFOVEU1Vm1WeWFXWnBZMkYwYVc5dVMyVjVNakF4T0NKOUxIc2lZMjl1ZEhKdmJHeGxjaUk2SWlJc0ltbGtJam9pSTJ0bGVTMHlJaXdpY0hWaWJHbGpTMlY1UW1GelpUVTRJam9pUkdoYWNUTk9OM2htVUhCTVkwVkZaVFJ0ZWxONVYyUm1VVFowVVhOeGFXdFdOVVprVVVoVldtcFNlV1FpTENKMGVYQmxJam9pV0RJMU5URTVTMlY1UVdkeVpXVnRaVzUwUzJWNU1qQXhPU0o5WFN3aWMyVnlkbWxqWlNJNlczc2lhV1FpT2lJME5XUTRNR1psWlMxa1lUVTRMVFJrWWpjdFlUa3lZeTFrT0ROa09XRXlaV1l6TVRRaUxDSndjbWx2Y21sMGVTSTZNQ3dpY21WamFYQnBaVzUwUzJWNWN5STZXeUlqYTJWNUxUSWlYU3dpYzJWeWRtbGpaVVZ1WkhCdmFXNTBJam9pYUhSMGNEb3ZMMnh2WTJGc2FHOXpkRG8wT1Rrd09DSXNJblI1Y0dVaU9pSkVTVVJEYjIxdFRXVnpjMkZuYVc1bkluMWRMQ0poZFhSb1pXNTBhV05oZEdsdmJpSTZXeUlqYTJWNUxURWlYU3dpWVhOelpYSjBhVzl1VFdWMGFHOWtJanBiSWlOclpYa3RNU0pkTENKclpYbEJaM0psWlcxbGJuUWlPbHNpSTJ0bGVTMHlJbDBzSW1OeVpXRjBaV1FpT2lJeU1ESXlMVEF6TFRJNFZERXpPak0xT2pNeExqRTJNelUwT0Mwd05Eb3dNQ0lzSW5Wd1pHRjBaV1FpT2lJeU1ESXlMVEF6TFRJNFZERXpPak0xT2pNeExqRTJNelUwT0Mwd05Eb3dNQ0o5Iiwid2hlbiI6IjIwMjItMDMtMjhUMTM6MzU6MzEuMTYzODM0LTA0OjAwIn1d",
```

Comment:
can we stick to `did:example:123`, and provide the did document so that keys and service endpoint requirements are explicit

Reply:
done